01-shapefile-attributes.Rmd

```diff
@@ -147,34 +148,21 @@ extent(point_HARV)
 point_HARV
 ```
 
-
 ##About Shapefile Attributes
 Shapefiles often contain an associated database or spreadsheet of values called
-`attributes` that describe the vector features in the shapefile. You can think
+**attributes** that describe the vector features in the shapefile. You can think
 of this like a spreadsheet with rows and columns. Each column in the spreadsheet
-is an individual `attribute` that describes an object (sometimes called
-variables). Shapefile `attributes` include measurements that correspond to the
+is an individual **attribute** that describes an object (sometimes called a
+variable). Shapefile attributes include measurements that correspond to the
 geometry of the shapefile features.
 
-For example, the Roads shapefile (`lines_HARV` object) contains an attribute
-called `TYPE`. Each line in the shapefile has an associated `TYPE` which 
-describes the type of road (road, footpath, boardwalk, etc). 
-
-<figure>
-    <a href="{{ site.baseurl }}/images/spatialVector/Attribute_Table.png">
-    <img src="{{ site.baseurl }}/images/spatialVector/Attribute_Table.png"></a>
-    <figcaption>The shapefile format allows us to store attributes for each feature
-    (vector object) stored in the shapefile. The attribute table, is similar to 
-    a spreadsheet. There is a row for each feature. The first column contains the 
-    unique ID of the feature. We can add additional columns that describe the 
-    feature. 
-    Image Source: National Ecological Observatory Network (NEON) 
-    </figcaption>
-</figure>
+For example, the `HARV_Roads` shapefile (`lines_HARV` object) contains an
```

Comment:
It is still in this location in the file on my computer?  Not sure what is going on here.  Will check again after next commit.